hedera-node/src/test/java/com/hedera/services/state/expiry/ExpiryManagerTest.java

```diff
@@ -113,15 +113,37 @@ private AccountID asAccount(long num) {
 		return IdUtils.asAccount(String.format("0.0.%d", num));
 	}
 
+	@Test
+	void restartClearsExistingState() {
+		// setup:
+		long oldExpiry = 2_345_678L;
+		AccountID payer = IdUtils.asAccount("0.0.2");
+		txnHistories = mock(Map.class);
+
+		// given:
+		subject = new ExpiryManager(recordCache, txnHistories);
+		// and:
+		subject.trackRecord(payer, oldExpiry);
+		// and:
+		givenAccount(2, new long[] { expiry });
+		// and:
+		given(txnHistories.computeIfAbsent(any(), any())).willReturn(new TxnIdRecentHistory());
+
+		// when:
+		subject.restartTrackingFrom(accounts);
+
+		// then:
+		verify(recordCache).reset();
+		verify(txnHistories).clear();
+	}
+
 	@Test
 	public void resumesTrackingAsExpected() {
```

Comment:
Should be renamed into `restartsTrackingAsExpected`

Reply:
Fixed